usaspending_api/common/views.py

```diff
@@ -45,7 +45,7 @@ def post(self, request, *args, **kwargs):
 
 class DetailViewSet(viewsets.ReadOnlyModelViewSet):
     """
-    Handles the views for endpoints that request a detailed
+    Handles the v2 for endpoints that request a detailed
```

Comment:
This viewset isn't only used for v2 so the description should probably be reverted